modules/textual_inversion/textual_inversion.py

```diff
@@ -238,6 +244,10 @@ def load_from_dir(self, embdir):
                     print(traceback.format_exc(), file=sys.stderr)
                     continue
 
+        if not embeddings_pickle_file.exists():

```

Comment:
You only dump embeddings when the pickle files does not exist. When would you update the embeddings?